src/django/contricleaner/lib/serializers/row_serializers/row_clean_field_serializer.py

```diff
@@ -8,7 +8,12 @@ def __init__(self, field: str, new_field: str) -> None:
         self.new_field = new_field
 
     def validate(self, row: dict, current: dict) -> dict:
-        clean_value = clean(row[self.field])
+        field = row.get(self.field)
+
+        if not field:
```

Comment:
This change seems to be wrong. If there's no value for clean data, it should return error.